src/feature/DeckBuilder/index.tsx

```
@@ -700,25 +711,26 @@ export const DeckBuilder = ({ implementedIds }: DeckBuilderProps) => {
   }, [deck]);
 
   const handleSaveDeck = useCallback(
-    (title: string, isMainDeck: boolean) => {
+    async (title: string, isMainDeck: boolean) => {
       if (deck.length === 40) {
-        LocalStorageHelper.saveDeck(title, deck, jokers, isMainDeck);
-        setCurrentDeckTitle(title);
+        try {
+          const savedDeck = await saveDeckToStorage(title, deck, jokers, isMainDeck);
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)